domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1609,79 +1609,113 @@ public TotaledResults(ResultSet set, Integer total) {
    * @param replyHandler
    */
   @SuppressWarnings({"unchecked", "squid:S00107"})
-  public <T> void streamGet(String table, T entity, String fieldName, CQLWrapper filter, boolean returnIdField,
-      String distinctOn, Handler<T> streamHandler, Handler<AsyncResult<Void>> replyHandler) {
+  public <T> void streamGet(String table, T entity, String fieldName,
```

Comment:
Can you add javadoc whether this is legacy?
Is this method deprecated? If yes add deprecation javadoc and annotation.